src/Microsoft.AspNetCore.Razor.Language/RazorDiagnosticFactory.cs

```diff
@@ -316,6 +316,32 @@ public static RazorDiagnostic CreateTagHelper_InvalidTargetedAttributeName(strin
             return diagnostic;
         }
 
+        internal static readonly RazorDiagnosticDescriptor TagHelper_CodeBlocksNotSupportedInAttributes =
+            new RazorDiagnosticDescriptor(
+                $"{DiagnosticPrefix}3011",
+                () => LegacyResources.TagHelpers_CodeBlocks_NotSupported_InAttributes,
```

Comment:
@ajaybhargavb  left these in `LegacyResources` since you said that we'd do one pass to remove all of the `LegacyResources`.